cdap-integration-test/src/main/java/co/cask/cdap/test/IntegrationTestManager.java

```diff
@@ -404,6 +404,12 @@ private File createModuleJarFile(Class<?> cls) throws IOException {
   }
 
   @Override
+  public void deleteDatasetInstance(NamespaceId namespaceId, String datasetInstanceName) throws Exception {
```

Comment:
Please check if it is possible to add a test without adding these extra methods to TestBase. An equivalent of this should be possible using the DatasetAdmin.drop() API.